packages/KTable/__snapshots__/KTable.spec.js.snap

```diff
@@ -4,16 +4,16 @@ exports[`KTable has hover class when passed 1`] = `
 <table class="k-table has-hover">
   <thead>
     <tr>
-      <th>
+      <th class="sortable ">
         Name
       </th>
-      <th>
+      <th class=" ">
```

Comment:
looks like we're injecting a random space here in the class. we should trim the string or change `"sortable"` to `" sortable"` (prefixed with a space) in the :class definition.